src/langcheck/metrics/metric_value.py

```diff
@@ -162,9 +163,13 @@ def __post_init__(self):
         if self.threshold_op not in operators:
             raise ValueError(f'Invalid threshold operator: {self.threshold_op}')
 
+        if self.threshold is None:
+            raise ValueError("A threshold of `None` is not supported.")
+
+        # Set the result to `False` if the metric value is `None`
         self._threshold_results = [
             operators[self.threshold_op](x, self.threshold)
-            for x in self.metric_values
+            if x is not None else False for x in self.metric_values
```

Comment:
Maybe at least for `!==` the value shouldn't be `False`...?

Reply:
hmm but the way that we frame it is that `True` means that it's a "pass", and I don't think `None` should ever be a "pass"? E.g. if the user writes `toxicity != 1`, I think a `None` value should fail that check

(Maybe we should output a warning any time a user calls a comparison operator on a MetricValue with Nones?)

Reply:
> output a warning any time a user calls a comparison operator on a MetricValue with Nones

That sounds good to me, could you work on that?

Reply:
Added the warning!